src/main/scala/com/microsoft/sqlserver/jdbc/spark/utils/BulkCopyUtils.scala

```diff
@@ -240,65 +270,88 @@ object BulkCopyUtils extends Logging {
         val dfCols = df.schema
 
         val tableCols = getSchema(rs, JdbcDialects.get(url))
+        val computedCols = getComputedCols(conn, dbtable)
+
         val prefix = "Spark Dataframe and SQL Server table have differing"
 
-        assertIfCheckEnabled(dfCols.length == tableCols.length, strictSchemaCheck,
-            s"${prefix} numbers of columns")
+        if (computedCols.length == 0) {
+            assertIfCheckEnabled(dfCols.length == tableCols.length, strictSchemaCheck,
+                s"${prefix} numbers of columns")
+        } else if (strictSchemaCheck) {
```

Comment:
Good to have this check into a different function and clarify what exactly is being done here - Checking that nr of non computed cols in DF is same as that of non-computed cols in table

Reply:
Done. I created a function dfComputedColCount to get computed column number in df, then added more comments.